app/src/main/convert.js

```diff
@@ -37,7 +37,16 @@ export default async function (exportOptions) {
     }
   });
 
-  app.kap.mainWindow.send('export-progress', {text: ''});
+  ipcMain.on('cancel-export', () => {
+    convertProcess.cancel();
+  });
 
-  return outputPath;
+  return convertProcess
+    .then(() => {
```

Comment:
Use `await`

Reply:
@sindresorhus how to use await here with the cancel thing?

Reply:
Ah I got it